### DISCUSSION TITLE
model:   User|Admin

### DISCUSSION THREAD

**Discussion by jm61** [pocketbase/pocketbase#653] [Q&A]
Hello to all,
I try to have a ternary condition following this: "model:   User|Admin" but when I execute "client.authStore.model", it returns me the complete user and profile details... I can't get this detail only from "BaseAuthStore", I should miss something...
I know I could check the presence of the "verified" field in the reply but direct model would be more efficient :-) 
Regards.

**Comment by ganigeorgiev** [comment]
I'm not sure that I understand the question. Are you working on a custom AuthStore?

`client.authStore.model` currently returns a union type `User|Admin|null`.

You can check against the model instance by doing something like:

```js
import PocketBase, { User, Admin } from "pocketbase";

const client = new PocketBase(...);

...

if (client.authStore.model && client.authStore.model instanceof User) {
    console.log("user...")
}
```

**Reply by jm61** [reply]
I'm using standard authStore. I will use your above snippet to detect used model, thanks for your help.
